### PR TITLE
User registration email not validated #720

### PR DESCRIPTION
**Pull request by SatyaSandeep51**
User after registering with an Email gets the verification link to his Mailbox. By clicking on the verification link he/she will be redirected to the verifying page and from there user can go to login after the successful verification. Used a sample email and app specific password to generate emails to the registered user. 
Email Id : sampleemail304@gmail.com
Password : Welcome@123
App Specific Password : mecv erfk mosu fvji 

Modified Files :
-- src/Modules/SimplCommerce.Module.Core/Areas/Core/Controllers/AccountController.cs
-- src/Modules/SimplCommerce.Module.EmailSenderSmtp/EmailSender.cs

### CODE REVIEW

**Comment by hishamco** [comment]
Please resolve the conflicts

File: src/Modules/SimplCommerce.Module.EmailSenderSmtp/EmailSender.cs [hunk 1]
```diff
@@ -14,8 +14,8 @@ public class EmailSender : IEmailSender
         public EmailSender(IConfiguration config)
         {
             _emailConfig.SmtpServer = config.GetValue<string>("SmtpServer");
-            _emailConfig.SmtpUsername = config.GetValue<string>("SmtpUsername");
-            _emailConfig.SmtpPassword = config.GetValue<string>("SmtpPassword");
+            _emailConfig.SmtpUsername = "sampleemail304@gmail.com";
```
Comment (afernandes) [inline src/Modules/SimplCommerce.Module.EmailSenderSmtp/EmailSender.cs:17]: It is not recommended to use a fixed email.
The email must be configurable.

File: src/SimplCommerce.WebHost/appsettings.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.;Database=SimplCommerce;Trusted_Connection=True;TrustServerCertificate=true;MultipleActiveResultSets=true"
+    "DefaultConnection": "server=DotNetFSD\\SQLEXPRESS;database = SimplCommerce;user id = sa; password = pass@123;trustservercertificate = true;MultipleActiveResultSets=true"
```
Comment (afernandes) [inline src/SimplCommerce.WebHost/appsettings.json:3]: ???

**Review by afernandes — CHANGES_REQUESTED** [review]
Review changed settings

**Comment by hishamco** [comment]
Closed while there's no respond from the author

Feel free to reopen the PR again

**Pull request closed without merging**